_pages/deliver/measure.md

```diff
@@ -32,3 +33,9 @@ Interpreting metrics throughout the project can be tricky (and subjective). Howe
 ## Incorporation: What to do next
 
 - Periodically reevaluate your prioritized backlog, roadmap, and strategy based on the results of your measurements to ensure you’re on a path that will deliver good value to users.
+
+## Resources
```

Comment:
This section hasn't been completed yet.  Do you think it matters that it's a partial list (doesn't point to any references on user testing, for example), or that the format is different from the other resource sections?  I can go either way.  I'd probably slightly lean to it's better to include them if we have them, and no one will likely care about the inconsistencies.

Reply:
I think it's worth sharing the two resources we have for now even if it's incomplete.